tests/functional/monitoring/prometheus/alerts/test_rgw.py

```diff
@@ -38,7 +40,10 @@ def test_rgw_unavailable(measure_stop_rgw, threading_lock):
             "Please check Ceph cluster health or RGW connection."
         )
     else:
-        target_msg = "Cluster Object Store is in unhealthy state. Please check Ceph cluster health."
+        target_msg = (
+            "Cluster Object Store is in unhealthy state or number of ready replicas for "
+            "Rook Ceph RGW deployments is less than the desired replicas in namespace:cluster openshift-storage:."
```

Comment:
`openshift-storage` should not be hardcoded. Lets pick it up from config.ENV_DATA

Reply:
ack, used `constants.OPENSHIFT_STORAGE_NAMESPACE`